conjur/login/login_controller.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginController module
+
+This module is the controller that facilitates all login actions
+required to successfully configure the netrc
+"""
+
+# Builtins
+import getpass
+import logging
+import sys
+
+from conjur.init import ConjurrcData
+from conjur.utils import Utils
+import conjur.constants
+
+class LoginController:
+    """
+    LoginController
+
+    This class represents the Presentation Layer for the LOGIN command
+    """
+    password = None
+    credential_data = None
+    login_logic = None
+    ssl_verify = None
+
+    def __init__(self, ssl_verify, password, credential_data, login_logic):
+        self.ssl_verify = ssl_verify
+        if self.ssl_verify is False:
+            Utils.insecure_message()
+
+        self.password = password
+        self.credential_data = credential_data
+        self.login_logic = login_logic
+
+    def load(self):
+        """
+        Method that facilitates all method calls in this class
+        """
+        self.get_username()
+        self.password = self.get_password()
+        loaded_conjurrc = self.load_conjurrc_data()
+        self.get_api_key(self.ssl_verify, loaded_conjurrc)
+        # pylint: disable=logging-fstring-interpolation
+        logging.debug(f"Saving credentials to '{conjur.constants.DEFAULT_NETRC_FILE}'")
+        self.login_logic.save(self.credential_data)
+        # pylint: disable=logging-fstring-interpolation
+        logging.debug("Credentials successfully written to " \
+                      f"'{conjur.constants.DEFAULT_NETRC_FILE}'")
+        sys.stdout.write("Successfully logged in to Conjur!\n")
```

Comment:
Use print.
Also, the exclamation mark is not needed, and looks less professional, IMO (But it's up to the PM to decide).
```suggestion
        print("Successfully logged in to Conjur")
```

Reply:
Okay 👌 
I don't want to change to print because I think we should stick to a single way of printing to STDOUT. This change will need to be changed in a bunch of areas which I don't want to be part of this PR